shared/src/commonMain/kotlin/com.macaosoftware.sdui.app/marketplace/amadeus/data/model/citycode/CityCodeHotel.kt

```diff
@@ -0,0 +1,13 @@
+package com.macaosoftware.sdui.app.marketplace.amadeus.data.model.citycode
+
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class CityCodeHotel(
+    @SerialName("data")
+    val `data`: List<Data>,
```

Comment:
I think is better to find another name. I am not a big fan of using `` in kotlin. Use `dataList` or `dataCollection` or `dataValues`

Reply:
You are right but going with the response name, because we will have a lot of models and different data. I think its okay to be going with this. If you want, Can refactor it